content/usage/getting_help.md

```diff
@@ -18,3 +18,9 @@ You can also post questions or comments to our [community forum](https://discour
 # Reporting Issues
 
 We unfortunately receive a large number of false positive issues. We ask that you discuss potential problems in our [community forum](https://discourse.drone.io) before creating github issues.
+
+# IRC channel 
+
+The (unofficial) drone IRC channel is present on Freenode in #drone-ci .
```

Comment:
The community provides an unofficial IRC channel which is available on Freenode at [#drone-ci](https://webchat.freenode.net/?channels=drone-ci).

Reply:
And maybe #drone.io should be a better channel ;)

Reply:
Maybe, but we already collected an astonishing amount of 4 really active users already (jk) :)

Can we consider #drone-ci good enough for now ?

Reply:
Even these 4 people can switch without a problem to match the naming

Reply:
I think Drone-CI is a better naming convention, having a dot in a IRC channel name is just really odd.

Reply:
Why is it odd? This is just your personal opinion. This project is named drone.io.

Reply:
Switched to #drone.io